app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -188,7 +188,12 @@
   };
 
   StaticAnalytics.prototype.setUserJourneyStage = function(pageType) {
-    this.setDimension(33, pageType);
+    // Track the stage of a user's journey through GOV.UK. If the page does not
+    // set a page type in a meta tag, default to "thing", which identifes a page
+    // as containing content rather than some kind of navigation.
+    var defaultUserJourneyStageValue = "thing";
+    var dimensionValue = (typeof pageType === "undefined") ? defaultUserJourneyStageValue : pageType;
```

Comment:
> Is there any reason in particular you're checking specifically for `undefined` rather than using `!pageType ?`

It was to be consistent with line 135, which checks for the presence of dimension values in general. But on reflection, it probably makes sense to just check for falsiness here because we know what kind of values to expect (`finding` or `thing` or some other non-empty value). Whereas line 135 can't make that kind of assumption - there might be a dimension for which an empty string is meaningful. I've updated it, thanks.

> Also can you please rename `pageType` to `journeyStage` (or similar)?

Done. I left it as `pageType` because it is a type of page, but if it looks like a mistake then it's worth changing.